src/librustc_mir/transform/qualify_consts.rs

```diff
@@ -660,6 +659,82 @@ impl<'a, 'tcx> Checker<'a, 'tcx> {
         }
     }
 
+    /// Check fn in const context with supplied feature gates
+    fn check_fn_in_const_ctx(&mut self, def_id: DefId) -> () {
```

Comment:
Why not just `check_fn`?  That seems to be more like how the other functions here are called.

Reply:
I have no opinion, this is what it was before I took over so I kept it. I'll rename it.